lms/djangoapps/grades/api/v0/views.py

```diff
@@ -0,0 +1,142 @@
+""" API v0 views. """
+import logging
+
+from django.http import Http404
+from edx_rest_framework_extensions.authentication import JwtAuthentication
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from rest_framework import status
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.generics import GenericAPIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from lms.djangoapps.ccx.utils import prep_course_for_grading
+from lms.djangoapps.courseware import courses
+from lms.djangoapps.grades.new.course_grade import CourseGradeFactory
+from openedx.core.lib.api.authentication import OAuth2AuthenticationAllowInactiveUser
+
+log = logging.getLogger(__name__)
+
+
+class CurrentGradeView(GenericAPIView):
+    """
+    **Use Case**
+
+        * Get the current grade for a specific user in a course.
+
+    **Example Request**
+
+        GET /api/grades/v0/course_grade/{course_id}/?username={username}
+
+    **GET Parameters**
+
+        A GET request must include the following parameters.
+
+        * course_id: A string representation of a Course ID.
+        * username: A string representation of an user's username.
+
+    **GET Response Values**
+
+        If the request for information about the course progress status
+        is successful, an HTTP 200 "OK" response is returned.
+
+        The HTTP 200 response has the following values.
+
+        * username: A string representation of an user's username passed in the request.
+
+        * course_id: A string representation of a Course ID.
+
+        * passed: Boolean representing whether the course has been passed
+
+        * percent: A float represent the overall grade for the course
+
+        * grade: A letter grade as defined in grading_policy (e.g. 'A' 'B' 'C' for 6.002x) or None
```

Comment:
Can we change `grade` to `letter_grade`?
Reason: "grade" has different semantics and multiple components.  So it's better to be specific here for forward compatibility.